tests/components/element-editor/basic-details/common/id.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import "jest-canvas-mock";
+import { IdField } from "src/components/element-editor/basic-details/common";
+import { TestHelper } from "src/util/test-helper";
+
+describe("Basic Details Id Component", () => {
```

Comment:
use the component name as the main test suite description

```suggestion
describe("IdField", () => {
```